umbraco-cloud/getting-started/umbraco-cloud-plans.md

```diff
@@ -18,30 +18,28 @@ Currently, all available Umbraco Cloud plans are utilizing P1V3 Azure App Servic
 
 To ensure stable performance of all websites hosted on Umbraco Cloud shared plans, soft and hard quotas were put in place. Quotas per site and the number of sites in the pool vary per Umbraco Cloud Plan.
 
-## Hard quotas for shared Umbraco Cloud Plans
+## Plan quotas for shared Umbraco Cloud Plans
 
-Whenever a CPU or RAM quota is met, the website will be restarted within a minute to ensure stable performance for all websites in the pool. Meeting a disk quota will result in errors whenever performing write operations to disk. No hard limits exist for TCP connections per site, besides the total amount of TCP connections available to all sites in the pool set at 1,920, attempting to open new TCP connections afterwards will result in errors.
+Whenever a CPU or RAM quota is met, the website will be restarted within a minute to ensure stable performance for all websites in the pool. Meeting a disk quota will result in errors whenever performing write operations to disk. No plan limits exist for TCP connections per site, besides the total amount of TCP connections available to all sites in the pool set at 1,920, attempting to open new TCP connections afterwards will result in errors.
 
-These are hard quotas, as they can't be broken for more than a minute and will immediately restrict the functionality of the site.
+These are hard plan quotas, as they can't be broken for more than a 5 minute period. When an cloud environment uses more CPU or memory as defined by the plan quota, the app service hosting the environment will restart. If the app service is restarted several times, we consider it as noisy neighbour and will to move it to dedicated environment to ensure other tenants in the shared pool aren't negatively affected.
 
 Umbraco Cloud Starter plan
 
-- CPU - 20%
-- RAM - 1500 MB
+- CPU - 20% (120 sec of CPU time for a 5 minute period)
+- Memory - 1500 MB (in private bytes)
 - Disk - 7800 MB
 
 Umbraco Cloud Standard plan
 
-- CPU - 40%
-- RAM - 1800 MB
+- CPU - 35% (210 sec of CPU time for a 5 minute period)
+- Memory - 1800 MB (in private bytes)
 - Disk - 9600 MB
 
 Umbraco Cloud Professional plan
 
-- CPU - 50%
-- RAM - 2000 MB
+- CPU - 50% (210 sec of CPU time for a 5 minute period)
+- Memory - 2000 MB (in private bytes)
 - Disk - 10400 MB
 
-## Quotas for the shared pool
-
-Whenever a pool hits above 95% usage of any resource disk, the busiest staging or development environment, that hasn't been moved previously will be moved to a different pool.
+You will be able to see the current and past CPU and 
```

Comment:
@sajumb something is missing here 🤔

Reply:
Yup, my bad. I have deleted the incomplete sentence and rearranged a bit in a new commit.